src/styles/each.scss

```diff
@@ -0,0 +1,7 @@
+@each $start-day, $switch in $week-days {
+  &--start-day--#{$start-day} {
```

Comment:
It doesn't work now. Move it to a calendar.scss file inside the `.calendar` selector